resources.yaml

```diff
@@ -285,3 +285,9 @@ deployment:
     start: 2023-11-20
     end: 2023-12-12
     group: training-plus-course-r
+    group: interactive
```

Comment:
That's [invalid YAML](https://github.com/usegalaxy-eu/vgcn-infrastructure/actions/runs/6769377781/job/18395669357) I am afraid... I could modify [synchronize.py](https://github.com/usegalaxy-eu/vgcn-infrastructure/blob/911aa982cc0080fdb741f09adcf7801ba63c7953/synchronize.py) so that it takes a list as well as single values.

Reply:
Is that even processed by the script? I thought it was used only by the userdata template.

Reply:
Sorry, that's true, the script does not care much what you put there. What would have to be modified to accept both single items and lists is the [userdata template](https://github.com/usegalaxy-eu/vgcn-infrastructure/blob/911aa982cc0080fdb741f09adcf7801ba63c7953/userdata.yaml.j2). Thanks a lot for reminding that.

Reply:
Upon thinking, I am not sure if it's that simple. 

1. Yes, it is simple to add a jinja2 template that would take in the list and creates `GalaxyGroup_<index or count or something> = <group>` and then adds these to the `STARTD_ATTRS`

    Something like:
    ``` 
    {% for item in group %}
    GalaxyGroup{{ loop.index }} = {{ item }}
    {% endfor %}
    
    STARTD_ATTRS = GalaxyTraining, GalaxyCluster, GalaxyDockerHack, {% for item in range(1, group | length + 1) %}{% if item > 1 %}, {% endif %}GalaxyGroup_{{ item }}{% endfor %}
    ```
2. However, doing so might break TPV, monitoring system, and cron jobs that rely on the `GalaxyGroup` attribute.
3. We can indeed name the 1st group `GalaxyGroup` and the rest with the convention `GalaxyGroup_<index or count or something>`; this would still affect TPV destinations or tools in the current form. 

These are some of the things that came to my mind. I may have missed other things.

Reply:
Yes exactly, the underlying issue is that our current system does not allow a machine to belong to multiple groups.

Reply:
The easiest solution for this single case may be to put the `submits_requirements` [here](https://github.com/usegalaxy-eu/infrastructure-playbook/blob/cb1ea170eae636e448db8eb0385e2bafad21f125/files/galaxy/tpv/interactive_tools.yml#L9) in a parenthesis and append ` || (GalaxyGroup == "training-plus-course-r")`.

Obviously this does not scale well with the number of trainings requiring interactive tools.

Reply:
Sorry we'd need to do more more stuff/it can be done in a simpler way, but hopefully you get where I am going.

Reply:
Not sure if we can reimplement and change everything with what's already going on. I think we can add the hack to Rstudio tool in TPV. However, we should make a note that we should add the required flexibility to our infrastructure in near future.

Reply:
That would be my preferred option as well, let's go with that.

Reply:
We close this one and just use the main queue :) ok?

Reply:
Sure, I hope that will work.

Reply:
I have pushed a new commit updating the resources for this training. Please check and feel free to merge.